### PR TITLE
README: community communication

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts <vbatts@kinvolk.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,15 @@ uses its [Code of Conduct](./CODE_OF_CONDUCT.md) for all its projects.
...
 Please read it and keep it in mind when participating in our projects.
 
+## Email and Chat
+
+- Email:
+  - Flatcar Community list: [flatcar-user](https://groups.google.com/g/flatcar-linux-user)
+  - General inquiries: hello@kinvolk.io
+- Chat:
+  - Community #[flatcar](irc://irc.freenode.org:6667/#flatcar) IRC channel on freenode.org
+  - Community [Lokomotive](https://kubernetes.slack.com/archives/C010B5JFB4L) channl on Kubernetes' Slack
```
Comment (joaquimrocha) [inline README.md:24]: Typo on `channl`:
```suggestion
  - Community [Lokomotive](https://kubernetes.slack.com/archives/C010B5JFB4L) channel on Kubernetes' Slack
```

**Review by joaquimrocha — CHANGES_REQUESTED** [review]
There is a typo.

Also, I feel maybe it'd be better to have always the `Contact` section in each project's README and just link there (less things to update).

**Comment by vbatts** [comment]
They can list their contact section also. This is to be a catch-all info.

**Comment by vbatts** [comment]
updated PTAL

**Pull request merged**